src/eval/tests/trie.rs

```diff
@@ -10,7 +10,7 @@ fn trie_lang() {
 
     install(s, &mut lang);
 
-    let expr = "(let ((trie (.lurk.trie.new)))
+    let expr = "(let ((trie (trie-new)))
```

Comment:
The only changes I would expect here is that if the default package is a user package that uses `.lurk`, then the reference may become just `trie.new`.

Reply:
Makes sense.